ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_random.py

```diff
@@ -272,3 +273,38 @@ def test_paddle_randint_like(
         high=high,
         dtype=dtype[0],
     )
+
+
+# multinomial
+@handle_frontend_test(
+    fn_tree="paddle.tensor.random.multinomial",
+    input_dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid", full=False),
+        min_value=0.25,
+        max_value=0.25,
+        min_dim_size=5,
+        max_dim_size=5,
+        shape=helpers.get_shape(
+            min_num_dims=1, max_num_dims=2, min_dim_size=2
+        ),
+    ),
+)
```

Comment:
We need to generate strategies for `num_samples` and `replacement` arguments as well